JDI.Light/JDI.Light/Elements/Composite/Alerts.cs

```diff
@@ -3,9 +3,9 @@
 
 namespace JDI.Light.Elements.Composite
 {
-    public class Alert : UIElement
+    public class Alerts : UIElement
```

Comment:
@RomanIovlev , do we really need to use `Alerts` insetead of `Alert` here?
It's a single class represinting single element.